lib/Dropdown/DropdownAction.js

```diff
@@ -15,11 +15,15 @@ const DropdownAction = (
     hideAfterPerformingAction,
     selected,
     overflow,
-    disabled
+    disabled,
+    ...props
   },
   context
 ) => {
-  const { toggleShowContent, showContent } = context[GATHER_UI_DROPDOWN];
+  const {
```

Comment:
How come this change was required? The context should always exist. Otherwise, the action cannot function as expected.

Reply:
In UserList we don't render a wrapping Dropdown, only the groups and actions